frontend/pulse3d/src/serviceWorker.js

```diff
@@ -33,6 +31,7 @@ const tokens = {
 const setTokens = ({ access, refresh }) => {
   tokens.access = access.token;
   tokens.refresh = refresh.token;
+  setLogoutTimer();
```

Comment:
so this is where I was saying just clears the timeout and doesn't ensure that the `ClientSource.postMessage({ logout: true });` gets called inside the logout. Do you think that's something we should worry about or no?

Reply:
I think `setLogoutTimer` does that, do you mean when the timer gets cleared?

Reply:
Yes. I'll try to investigate the order of requests made next time it happens, but clearing the timeout doesn't ensure that the client enforces the auto logout

Reply:
and the only time the logout message gets sent to _app.js is inside the timeout callback

Reply:
Also, my bad, I meant to addd this comment to:
`const clearTokens = () => {
  clearLogoutTimer();
};`